src/NUnitFramework/tests/Internal/TestExecutionContextTests.cs

```diff
@@ -137,7 +137,25 @@ public void Cleanup()
         }
 #endif
 
-        #endregion
+#if !PORTABLE && !NETSTANDARD1_6
```

Comment:
This will also be a valid test for NETSTANDARD1_6 after #2010 is merged, I think. We should remember to enable it. 👍